fendermint/app/src/cmd/genesis.rs

```diff
@@ -189,9 +191,17 @@ where
 fn into_tendermint(genesis_file: &PathBuf, args: &GenesisIntoTendermintArgs) -> anyhow::Result<()> {
     let genesis = read_genesis(genesis_file)?;
     let genesis_json = serde_json::to_value(&genesis)?;
+    // Fendermint chainIDs can only be 50 bytes long. To avoid
+    // arbitrary chain names from exceeding the limit, we compute
+    // the chainID as the 54 most significant bytes of the SHA256
```

Comment:
Where are the other 4 bytes?

Reply:
This is a typo, it should say 50 in both, I will update accordingly. Fixed!